### DISCUSSION TITLE
如何对指定的域名，返回固定的IP

### DISCUSSION THREAD

**Discussion by dev4u** [LEXUGE/dcompass#82] [Q&A]
请教一下，我需要对几个域名，返回指定的静态IP地址。这个要怎么写配置呢？

**Comment by LEXUGE** [comment]
很抱歉，目前的版本不支持复写静态 IP。之前的版本支持，但是因为底层重构就没有支持了。

目前有计划在未来的版本中实现类似功能

**Reply by dev4u** [reply]
那我现在先和smartdns搭配使用
